src/schema.py

```diff
@@ -159,7 +159,7 @@ def resolve_account_info(self, info, session_id=None):
         name = location
 
       new_transaction = {
-          'amount': txn['amount'],
+          'amount': "{:.2f}".format(int(txn['amount'])),
```

Comment:
Doesn't casting to `int` lose the precision of the float?

Reply:
Yep, you're right